ansible/slurm.yml

```
@@ -61,6 +61,11 @@
   tags:
     - openhpc
   tasks:
+    - include_role:
+        name: topology
+      # Gated on topology group having compute nodes but role also
+      # needs to run on control and login nodes
+      when: appliances_mode == 'configure' and (groups['topology'] | length) > 0
```

Just a style thing, can we do:
```suggestion
      when:
        - appliances_mode == 'configure'
        - groups['topology'] | length > 0
```